### PR TITLE
Updated extras API and NFTs endpoints

### PR DESCRIPTION
**Pull request by razvantomegea**

### CODE REVIEW

File: src/redux/endpoints/nfts.endpoint.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ const nftsEndpoints = RootApi.injectEndpoints({
         baseURL: getBaseURL(),
         url: `/${ACCOUNTS_ENDPOINT}/${props.address}/${NFTS_ENDPOINT}`,
         method: 'GET',
-        params: { size: MAX_API_SIZE, ...props }
+        params: { excludeMetaESFDT: true, size: MAX_API_SIZE, ...props }
```
Comment (MiroMargineanu) [inline src/redux/endpoints/nfts.endpoint.ts:22]: As far as I know, the parameter should be called `excludeMetaESDT`? Looks like a typo here.

**Review by MiroMargineanu — CHANGES_REQUESTED** [review]

File: src/redux/endpoints/nfts.endpoint.ts (hunk 1 above)
Comment (razvantomegea) [reply src/redux/endpoints/nfts.endpoint.ts:22]: Correct

Comment (MiroMargineanu) [reply src/redux/endpoints/nfts.endpoint.ts]: Sweet. 🔥

**Review by MiroMargineanu — APPROVED** [review]

File: src/components/AxiosInterceptor/helpers/useSetNativeAuthInterceptors.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const useSetNativeAuthInterceptors = () => {
 
         let bearerToken = newToken;
         const hasAllowedURLs =
-          config.baseURL?.includes('extras-api.multiversx.com') ||
+          config.baseURL?.includes('extras-lite-api.multiversx.com') ||
```
Comment (arhtudormorar) [inline src/components/AxiosInterceptor/helpers/useSetNativeAuthInterceptors.ts:52]: nu va mai exista URL-ul vechi?

File: src/pages/Dashboard/widgets/Account/Account.tsx [hunk 3]
```diff
@@ -66,6 +66,7 @@ export const Account = () => {
             >
               Open in Explorer
             </a>
+            {isSovereign && <Faucet />}
```
Comment (arhtudormorar) [inline src/pages/Dashboard/widgets/Account/Account.tsx:69]: nu o sa mai avem pe devnet?

File: src/components/AxiosInterceptor/helpers/useSetNativeAuthInterceptors.ts (hunk 2 above)
Comment (razvantomegea) [reply src/components/AxiosInterceptor/helpers/useSetNativeAuthInterceptors.ts:52]: We will migrate to a public extras API, yes

File: src/pages/Dashboard/widgets/Account/Account.tsx (hunk 3 above)
Comment (razvantomegea) [reply src/pages/Dashboard/widgets/Account/Account.tsx:69]: No, only sovereign

**Review by arhtudormorar — APPROVED** [review]

**Pull request merged**